systems/robotInterfaces/Side.h

```diff
@@ -0,0 +1,33 @@
+#ifndef SYSTEMS_CONTROLLERS_SIDE_H_
+#define SYSTEMS_CONTROLLERS_SIDE_H_
+
+#include <array>
+#include <string>
+
+// adapted from https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Type_Safe_Enum
+class Side
```

Comment:
I actually also like Side (and am using it extensively in the updated s1traj branch).